projects/PropertiesTwice/PropertiesTwice.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <ImplicitUsings>disable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <ProductName>Test project</ProductName>
+    <Authors>Corniel Nobel, Wesley Baartman</Authors>
```

Comment:
`Authors` tag is delimited with semicolons (https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-target).